pkg/ingester/ingester_test.go

```diff
@@ -2523,9 +2523,19 @@ func TestIngester_QueryStream(t *testing.T) {
 	const numSeries = 1000
 
 	for seriesID := 0; seriesID < numSeries; seriesID++ {
-		lbls := labels.FromStrings(labels.MetricName, "foo", "series_id", strconv.Itoa(seriesID))
-		req, _, _, _ := mockWriteRequest(t, lbls, float64(seriesID), int64(seriesID))
-		_, err = i.Push(ctx, req)
+		floatLbls := labels.FromStrings(labels.MetricName, "foo", "series_id", strconv.Itoa(seriesID))
```

Comment:
I think we can keep all series name equal to "foo", just add another "type" label with values "float", "histogram", "floathistogram". That will create 3 different metrics, since metrics are identified by all labels together.

Reply:
Good idea - done!